frontend/src/screens/learner.tsx

```diff
@@ -126,17 +133,29 @@ export const StudiesContainer = () => {
                     <SearchBar search={search} setSearch={setSearch} />
                 </Flex>
 
+                {/* Divider */}
```

Comment:
https://mantine.dev/core/divider/

do you want to just use this instead? consistent with the other uses in here